fw/http_stream.h

```diff
@@ -121,23 +123,27 @@ typedef enum {
  * @postponed		- head of skb list that must be sent
  *			  after sending headers for this stream;
  * @h_len		- length of headers in http2 response;
+ * @t_len		- length of trailer headers in http2 response;
  * @frame_length	- length of current sending frame, or 0
  *			  if we send some service frames (for
  *			  example RST STREAM after all pending data);
  * @b_len		- length of body in http2 response;
  * @is_blocked		- stream is blocked;
  * @state		- current stream xmit state (what type of
  * 			  frame should be made for this stream);
+ * @is_trailer_cont	- need to send trailer CONTINUATION frames;
  */
 typedef struct {
 	TfwHttpResp 		*resp;
 	struct sk_buff		*skb_head;
 	struct sk_buff		*postponed;
 	unsigned int		h_len;
+	unsigned int		t_len;
```

Comment:
This structure now has 5 bytes hole. please move `t_len` to avoid this
 `pahole -C TfwHttpXmit fw/tempesta_fw.ko`